dvc/fs/dvc.py

```diff
@@ -68,13 +68,17 @@ def open(  # type: ignore
             raise IsADirectoryError
 
         out = outs[0]
+
+        if not out.hash_info:
+            raise FileNotFoundError
+
```

Comment:
Is this covered already? The tests below seem to be for another change.

Reply:
Yeah, there is an existing tests below that tests for this and I wrote a new one for NoRemoteError.